source/components/tables/tbutils.c

```diff
@@ -257,7 +257,7 @@ AcpiTbCheckDsdtHeader (
  *
  * FUNCTION:    AcpiTbCopyDsdt
  *
- * PARAMETERS:  TableDesc           - Installed table to copy
+ * PARAMETERS:  TableIndex         - Index of installed table to copy
```

Comment:
```
- Index of ...
```
Seems not correct.
Has it been aligned to the parameter description column?
It should be a column number of multiple-4.

Normally I'll duplicate this line and change 1 of them so that I can align it correctly.